### PR TITLE
Fix websocket URLs by bringing back cleaner asWebsocketUrl cleaner

### PR DESCRIPTION
**Pull request by samholmes**

### CODE REVIEW

File: src/common/utxobased/engine/types.ts [hunk 1]
```diff
@@ -3,8 +3,14 @@ import { EdgeSpendInfo } from 'edge-core-js/types'
 
 import { Input, Output } from '../keymanager/utxopicker/types'
 
+// Normalizes a URL to always be a websocket URL
+export const asWebsocketUrl = (raw: unknown): string => {
+  const url = asString(raw)
+  return `${url}/websocket`
```
Comment (swansontec) [inline src/common/utxobased/engine/types.ts:9]: You could use the safer regexp approach to only add this if needed, as isn’t the deleted code.

**Review by swansontec — APPROVED** [review]
Approved, but we could keep the regexp, just inside the cleaner.

**Comment by paullinator** [comment]
dont merge

**Review by paullinator — CHANGES_REQUESTED** [review]
Please change implementation to not add `websocket` to the URL. Instead only add it when you're opening the socket. Otherwise this will get included in the server name when stored on disk

**Pull request merged**